packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -350,15 +351,36 @@ class _SummaryCardState extends State<SummaryCard> {
         ),
       ),
       alignment: Alignment.topLeft,
-      padding: const EdgeInsets.symmetric(vertical: SMALL_SPACE),
-      child: Center(
-        child: Text(
-          helper.getHeaderText(AppLocalizations.of(context)),
-          style: Theme.of(context)
-              .textTheme
-              .subtitle1!
-              .apply(color: helper.getHeaderForegroundColor(isDarkMode)),
-        ),
+      padding: const EdgeInsets.symmetric(
+          vertical: SMALL_SPACE, horizontal: SMALL_SPACE),
+      child: Row(
+        children: <Widget>[
+          const Icon(
+            Icons.settings,
+            color: Colors.transparent,
+          ),
+          Expanded(
+            child: Center(
+              child: Text(helper.getHeaderText(AppLocalizations.of(context)),
+                  style: Theme.of(context).textTheme.subtitle1),
+            ),
+          ),
+          IconButton(
+            padding: EdgeInsets.zero,
+            constraints: const BoxConstraints(),
```

Comment:
Is this supposed to be like that

Reply:
> Is this supposed to be like that

Just for removing the default padding 45 around the IconButton, I did not find a better approach 😅